charts/kyverno/crds/crds.yaml

```diff
@@ -3,6 +3,16 @@ kind: CustomResourceDefinition
 metadata:
   name: clusterpolicies.kyverno.io
 spec:
+  additionalPrinterColumns:
+  - JSONPath: .spec.background
+    description: Background provides choice for applying rules to existing resources.
+    name: Background
+    type: string
+  - JSONPath: .spec.validationFailureAction
+    description: ValidationFailureAction provides choice to enforce rules to resources
```

Comment:
update description to, "ValidationFailureAction controls if a policy failure should not disallow an admission review request (enforce), or allow (audit) and report an error."